File: app/serena_mcsv/mscv-user/src/main/java/psytobetter/user/mscv_user/model/UserActiveSession.java
package psytobetter.user.mscv_user.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Entity
@Table(name = "sesion_activa")
@Data
public class UserActiveSession {

    @Id
    @Column(name = "session_id")
    private Long sessionId;

    @Column(name = "id_usuario_activo", nullable = false)
    private Long activeUserId;
}

File: app/src/main/java/com/duocuc/serena/data/UserActiveSession.kt
package com.duocuc.serena.data

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "SesionActiva")
data class UserActiveSession(

    @PrimaryKey
    val sessionId: Int = 1,

    @ColumnInfo(name = "id_usuario_activo")
    val activeUserId: Int
)